### DISCUSSION TITLE
我无法付费怎么办？

### DISCUSSION THREAD

**Discussion by lufangdong** [getlantern/lantern-forum-cn#747] [Q&A]
我在大陆无法续费，目前已经无法使用，怎么办？

**Comment by MaxMKMax** [comment]
@lufangdong 您好. 请问您用的是什么支付方式呢? 我帮您问问看情况. 谢谢您对Lantern的支持.

**Comment by MaxMKMax** [comment]
@lufangdong 您好。我可以知道为什么您无法续订，因为您没有看到付款选项，或者是否存在阻止您付款的错误或付款限制? 您介意告诉我确切的问题并为我提供一些屏幕截图吗？太感谢了。

**Comment by lufangdong** [comment]
我用银联支付

Mike K ***@***.***> 于2024年8月2日周五 00:38写道：

> @lufangdong <https://github.com/lufangdong>
> 你好。我可以知道为什么您无法续订，因为您没有看到他们拥有的付款选项，或者是否存在阻止他们付款的错误或限制?
> 您介意告诉我确切的问题并为我提供一些屏幕截图吗？太感谢了。
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/getlantern/lantern-forum-cn/discussions/747#discussioncomment-10214762>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BKIJH4JDU4SJWZB3JZ5DRKDZPJP7FAVCNFSM6AAAAABL23MBQSVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMRRGQ3TMMQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> .com>
>


-- 

Thanks & Regards，



*Fangdong Lu**,   **卢方东*

**Reply by MaxMKMax** [reply]
@lufangdong 您好。我可以知道为什么您无法续订，因为您没有看到付款选项，或者是否存在阻止您付款的错误或付款限制? 您介意告诉我确切的问题并为我提供一些屏幕截图吗？太感谢了。
